components/script/dom/headers.rs

```diff
@@ -63,57 +64,44 @@ impl Headers {
 impl HeadersMethods for Headers {
     // https://fetch.spec.whatwg.org/#concept-headers-append
     fn Append(&self, name: ByteString, value: ByteString) -> ErrorResult {
-        // Step 1
```

Comment:
I think the "Step 1" etc comments can be quite helpful. It's unclear to me how some of the code here fits in with the steps described in https://fetch.spec.whatwg.org/#concept-headers-append.